sdk/python/feast/__init__.py

```diff
@@ -47,4 +48,5 @@
     "RedshiftSource",
     "RequestFeatureView",
     "SnowflakeSource",
+    "PostgreSQLSource",
```

Comment:
not sure we need this here - we don't do this for `SparkSource`

Reply:
It's true that it's not needed, but it allows PostgreSQLSource to be imported like this:
```python
# This
from feast import PostgreSQLSource

# VS this
from feast.infra.offline_stores.contrib.postgres_source import PostgreSQLSource
```

I don't have strong feelings about this. If you could confirm that you prefer the full path when importing PostgreSQLSource I'll remove it from the __init__ file.

Reply:
I think this is fine, probably better to add SparkSource and TrinoSource to this file too. Will do that in separate pr.

Reply:
I think we do prefer the full path here to let users know that this is a contrib module. We do want to move this probably eventually to full support, but would remove this for now.